### ISSUE TITLE
Blank Canvas: Issues with the Split Screen pattern

### ISSUE THREAD

**Issue by alaczek** [Automattic/themes#3282]
The Split Screen block pattern included in Blank Canvas has a thin grey border around the cover block. This is visible in the inserter, and once the pattern is selected in the editor, but not in the preview. I also tried switching to a different theme (Mayland) after I created a page with this pattern, and the border was still there in the editor.

## Steps to replicate
1. On a site with Blank Canvas active, create a post or a page.
2. Insert the Split Screen pattern. Note the border around the cover block in the inserter preview, and in the editor.
3. Preview the site. The border is not there.

I expected there would be no border around the cover block.

Inserter view:
![image](https://user-images.githubusercontent.com/4550351/108015487-7ceed700-7064-11eb-96a0-801f927075b0.png)

Editor:
![image](https://user-images.githubusercontent.com/4550351/108015564-9abc3c00-7064-11eb-8030-5b35b1da855c.png)

Preview:
![image](https://user-images.githubusercontent.com/4550351/108015596-a576d100-7064-11eb-9a79-6a9ecb0c99ef.png)

cc @kjellr

Found through my own testing.

**Comment by kjellr** [comment]
I noticed this too — I think it's a Gutenberg UI issue, since I can reproduce in other themes:

1. Add a Cover block
2. Give it a white background (or another very-light color)
3. Notice the gray outline around it in the editor. 

 @jasmussen or @jameskoster are you aware of any new efforts to add UI borders around bocks? I'm not sure if this is intentional or not.

**Comment by jameskoster** [comment]
I can't think of anything we've changed that would cause this. Any changes we've made to block outline treatments have been restricted to the Site Editor anyway, and I notice this is happening in the post editor as well.

It's a strange one, the "style" seems to be applied to `img.wp-block-cover__image-background`... when I remove that element the phantom border also vanishes. But I can't see where it's being applied in the first place... both `border` and `outline` on that class are set to `none`, and it is positioned absolutely to fill the container.

I wonder if that `img` itself tag is a bug here... why is an `img` being rendered when I haven't set a background image? 🤔 

TLDR: Seems to be a UI bug with the cover block in core.

**Comment by kjellr** [comment]
Cool, thanks. I've opened https://github.com/WordPress/gutenberg/issues/29082 to track this issue, and will close this bug in the meantime.
